lib/Conch/Controller/HardwareProduct.pm

```diff
@@ -45,9 +45,9 @@ sub find_hardware_product ($c) {
     # route restricts key to: sku, name, alias
     if (my $key = $c->stash('hardware_product_key')
             and my $value = $c->stash('hardware_product_value')) {
-        $c->log->debug("Looking up a HardwareProduct by identifier ($key = $value)");
+        $c->log->debug('Looking up a HardwareProduct by identifier ('.$key.' = '.$value.')');
```

Comment:
Places like this seem like a step backwards, we _lose_ clarify and simplicity by having to concatenate instead of interpolate the string.

Reply:
I find string interpolation to be confusing because it is less explicit -- sometimes things get interpolated unintentionally and produces surprising effects.

Reply:
I have no issue with only using double quotes when you want to explicitly interpolate. I find inline concatenation operators get in the way of the intent of  the string that's being logged or assigned or returned in the general case.